src/components/transactions/Transactions.vue

```diff
@@ -113,33 +109,28 @@ export default {
         })
       }
     },
-    refreshTransactions() {
+    async refreshTransactions() {
       if (!this.sharedState.accountAddress) {
         this.transactions = []
         return
       }
-      // Bring transactions from version 1
-      // we will remove this after some time running version 2
-      let activeAddressRsk2EthTxns = TXN_Storage.getAllTxns4Address(
+      /**
+       * Synchronize transactions from localstorage
+       * TODO: consider removing for later versions
```

Comment:
I'd maybe add this to a Github issue and/or a ticket in Jira tagged as technical debt (so we don't lose sight of it)